portfolio/src/main/webapp/index.html

```diff
@@ -5,37 +5,80 @@
   <meta charset="UTF-8">
   <title>My Portfolio</title>
   <link rel="stylesheet" href="style.css">
-  <script src="script.js"></script>
+  <script defer src="script.js"></script>
 </head>
 
 <body>
   <div id="content">
-    <h1 id="title">My Portfolio</h1>
-    <div class="subsection">
-      <h2>Links</h2>
-      <p>This section will have links to other profiles</p>
-    </div>
+    <h1 class="centered">My Portfolio</h1>
     <div class="subsection">
       <h2>About Me</h2>
-      <p>This is my portfolio.</p>
-      <p>My name is Kyle Insaurralde</p>
+      <p>My name is Kyle Insaurralde. I am a junior at UC Davis. My favorite programming language is python. For
+      fun, I like to play minecraft.</p>
```

Comment:
You can use 4 space indention in wrapped lines. See here: https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping